src/app/app.js

```diff
@@ -105,6 +102,9 @@ angular.module('prx.base',['prx'])
 .config(/* istanbul ignore next */
   function ($locationProvider) {
     $locationProvider.html5Mode(true);
+}).run(/* istanbul ignore next */
+  function (PrxAuth) {
+    PrxAuth.$checkLoggedIn();
```

Comment:
the `prx.base` module is not loaded during unit tests, so it's the place we should put stuff that really only makes sense when we're running in the context of a web browser, either for E2E tests or deployment. Doing our initial logged-in check here prevents us from having to stub the network request for literally every test in the system.